### PR TITLE
Remove minor references to ShARC from general pages.

### PR DESCRIPTION
**Pull request by willfurnass**

### CODE REVIEW

**Comment by tdjames1** [comment]
I made a pr to your branch with the refs I found!

**Comment by willfurnass** [comment]
I did some bad rebasing here; PR currently includes superfluous commits.  Will rectify....

**Comment by willfurnass** [comment]
Rebase-induced badness fixed.

File: hpc/modules.rst [hunk 1]
```diff
@@ -281,7 +239,7 @@ Then if you want to load these modules **in an interactive session or in a batch
 
     source /home/te1st/proj1/setup_env.sh
 
-If you want to run the job on Stanage, Bessemer and ShARC (which provide different software / module files) 
+If you want to run the job on Stanage and Bessemer (which provide different software / module files) 
 you could adapt your script to load different modules depending on which cluster you are using: ::
```
Comment (Carldkennedy) [inline hpc/modules.rst:243]:
```suggestion
you could adapt your script to load different modules depending on which cluster you are using:

.. code-block:: bash 
```

**Review by Carldkennedy — APPROVED** [review]
LGTM. One suggestion improves format of code block

**Pull request merged**